openjdk/playlist.xml

```diff
@@ -36,6 +36,28 @@
 			<group>openjdk</group>
 		</groups>
 	</test>
+	<test>
+		<testCaseName>jdk_osb</testCaseName>
+		<variations>
+			<variation>Mode150</variation>
+			<variation>Mode650</variation>
+			<variation>Mode1000</variation>
+		</variations>
+		<command>$(JAVA_COMMAND) -Xmx512m -jar $(Q)$(TEST_RESROOT)$(D)jtreg$(D)lib$(D)jtreg.jar$(Q) \
+	$(JTREG_BASIC_OPTIONS) $(CUSTOM_NATIVE_OPTIONS) -vmoptions:$(Q)-Xmx512m $(JVM_OPTIONS) $(VMOPTION_HEADLESS)$(Q) \
+	$(TIMEOUT_HANDLER) \
+	-w $(Q)$(REPORTDIR)$(D)work$(Q) \
```

Comment:
Whether to consider uniformly pointing the jtreg test report path to REPORTDIR
-r $(Q)$(REPORTDIR)$(D)report$(Q)